packages/web-react/src/components/Pagination/Pagination.stories.ts

```diff
@@ -0,0 +1,18 @@
+import { ComponentMeta } from '@storybook/react';
+import Pagination from './Pagination';
+
+export default {
+  title: 'Components/Pagination',
+  component: Pagination,
+  parameters: {
+    docs: {
+      description: {
+        component:
+          'This will render a pagination control with provided pages, a default current page of 1, and a chapter size of 5. When the user clicks on a page number or one of the chevron, the onPageChange callback function will be called with the new page number as its argument.',
```

Comment:
```suggestion
          'This will render a pagination control with provided pages, a default current page of 1, and a chapter size of 5. When the user clicks on a page number or one of the chevrons, the onPageChange callback function will be called with the new page number as its argument.',
```